requirements.txt

```diff
@@ -16,3 +16,12 @@ flask-talisman==1.0.0
 dash_auth==2.0.0
 arrow==1.3.0
 dash-leaflet==1.0.7
+# dash[diskcache] dependencies
+# from https://github.com/plotly/dash/blob/dev/requirements/diskcache.txt
+diskcache>=5.2.1
+multiprocess>=0.70.12
+psutil>=5.8.0
+dash-ag-grid>=31.3.0
+dash-iconify>=0.1.2
+dash-mantine-components>=0.15.1
+dash[diskcache]
```

Comment:
Can you explain why `dash[diskcache]` was added?

I tried doing this before and it didn't seem to work, which is why I manually added `diskcache`, `multiprocess`, and `psutil` above as a workaround, which I believe accomplishes the same thing

Whichever way we go, it seems like we should only need one or the other